examples/docker/nginx.docker-compose.yml

```diff
@@ -19,7 +19,6 @@ services:
       - INSECURE_SERVER=TRUE
       # NOTE!: Replace with your identity provider settings https://www.pomerium.com/docs/identity-providers.html
       # - IDP_PROVIDER=google
-      # - IDP_PROVIDER_URL=https://accounts.google.com
```

Comment:
Could we replace this reference with another provider this would be relevant for? Maybe azure as you did below in the docs?